src/modules/flutterwave/helper/payment-plan.ts

```diff
@@ -0,0 +1,5 @@
+export class PaymentPlan {
+  async getPaymentPlanById(plan_id: string) {
```

Comment:
What does this do? I suppose you can use it to retrieve the payment in order to update it's status like I mentioned above

Reply:
Not using it yet